[ko] Update outdated files in dev-1.22-ko.4 (M1-M11)

jihoon-seo opened the pull request:
- Related issue: #30846

> * [ ]  M1.  `content/en/_index.html`  | 2(+XS) 2(-)
> * [ ]  M2.  `content/en/community/_index.html`  | 1(+XS) 0(-)
> * [ ]  M3.  `content/en/docs/concepts/architecture/cloud-controller.md`  | 2(+XS) 2(-)
> * [ ]  M4.  `content/en/docs/concepts/architecture/nodes.md`  | 11(+S) 6(-)
> * [ ]  M5.  `content/en/docs/concepts/cluster-administration/addons.md`  | 6(+XS) 1(-)
> * [ ]  M6.  `content/en/docs/concepts/cluster-administration/networking.md`  | 1(+XS) 50(-)
> * [ ]  M7.  `content/en/docs/concepts/configuration/manage-resources-containers.md`  | 8(+XS) 7(-)
> * [ ]  M8.  `content/en/docs/concepts/configuration/overview.md`  | 1(+XS) 1(-)
> * [ ]  M9.  `content/en/docs/concepts/configuration/secret.md`  | 1(+XS) 1(-)
> * [ ]  M10.  `content/en/docs/concepts/containers/container-lifecycle-hooks.md`  | 1(+XS) 1(-)
> * [ ]  M11.  `content/en/docs/concepts/containers/images.md`  | 68(+M) 0(-)

/language ko

seokho-son commented:
@jihoon-seo 
dev-1.22-ko.4 netlify 프리뷰 설정을 깜빡했네요;;
혹시 임의의 commit 추가(향후 스쿼싱될) 또는 PR 재오픈 가능하실까요? ^^

jihoon-seo commented:
@seokho-son 완료했습니다~~
(메인 페이지의 한국어를 수정하는 커밋입니다.
Update outdated files 하는 내용은 아니지만, bug fix 입니다.. 😊)

ClaudiaJKang commented:
@jihoon-seo 수고하셨습니다!
/lgtm

jihoon-seo commented on content/ko/docs/concepts/containers/images.md:
```diff
@@ -258,6 +265,73 @@ kubectl describe pods/private-image-test-1 | grep 'Failed'
+    { term }
+
+term:
+    '*'         matches any sequence of non-Separator characters 구분자가 아닌 모든 문자와 매치됨
+    '?'         matches any single non-Separator character 구분자가 아닌 문자 1개와 매치됨
```
영문 지우는 것을 놓쳤습니다 😅

```suggestion
    '*'         구분자가 아닌 모든 문자와 매치됨
    '?'         구분자가 아닌 문자 1개와 매치됨
```

jihoon-seo commented:
/label tide/merge-method-squash

seokho-son commented:
@jihoon-seo `label tide/merge-method-squash` 을 적극적으로 사용해도 될지 확신이 없네요. 예전에 SIG-Docs에 문의해봤을 때는 어쩔 수 없는 경우에 사용하고 있다고 했었어요. 다시 한번 파악해볼 필요가 있겠습니다. ^^

jihoon-seo commented:
@seokho-son K8s Slack에서 "merge method squash" 로 검색해 보니
관련 thread들이 있고..
그 중에서 이런 thread가 있네요 😊

https://kubernetes.slack.com/archives/C5P3FE08M/p1635345481012900
("merge-method-squash를 default로 하자" 라는 메시지로 시작된 thread라서
본 건 ("label tide/merge-method-squash 을 적극적으로 사용해도 될지") 과는 살짝 다르기는 하지만
참고는 할 수 있겠습니다 😊)

예를 들어
https://github.com/kubernetes/kubernetes/pull/105855
이 PR은 6개의 커밋으로 구성되어 있고
`label tide/merge-method-squash` 이 붙어서

https://github.com/kubernetes/kubernetes/commit/3c87c43ceff6122637c8d8070601f7271026360e
이 커밋으로 squash 되었습니다.
- Squashed commit의 message = PR의 제목;
- Squashed commit의 additional message = 6개의 커밋들의 커밋 messages;

![image](https://user-images.githubusercontent.com/46767780/145751913-a7529328-f699-48bf-9c93-31d012375e54.png)

---

위의 Slack thread 에서 이런 대화가 오갔습니다.

> yeah, I prefer intentional squashing by the author as well

> +1
The label, imo is used only in some of the cases where the aggregate commit message would make sense (ideally if you have multiple commits making changes to the same file maybe)

> When Prow squashes, it doesn't squash-then-merge (which is what happens if the author squashes). A single commit goes onto the main branch with no clear indication of the merge bases that the commit represents.
I definitely don't want that kind of squash by default.

> auto-squash nukes GPG (and maybe DCO) from commits. so a contributor might say: "you removed my signature from my contributions, so please revert them".

> I don't think the tide/merge-method-squash label should be made default on all PRs. IMHO, it encourages bad commit practices and eventually when those commits are squashed, the message of the commit being merged has text which may not be meaningful to the context as [pohly] has shown examples of above.
I think it's prudent on part of reviewers to ask the PR author to meaningfully squash/reorganize the commits as is needed by that specific context.

---

다른 thread에서도 `label tide/merge-method-squash` 이 언급되었습니다.
https://kubernetes.slack.com/archives/C1TU9EB9S/p1637020967168200?thread_ts=1635346631.051100&cid=C1TU9EB9S
> yep, I have the same feeling. We need to state it somewhere so nobody will take it personally.  /label tide/merge-method-squash  is good, but may not be ideal and we need to state that in k/k manual squash is preferred unless it’s impossible.

---

위와 같은 상황을 이해했으며
또한
> `label tide/merge-method-squash` 을 적극적으로 사용해도 될지 확신이 없네요. 예전에 SIG-Docs에 문의해봤을 때는 어쩔 수 없는 경우에 사용하고 있다고 했었어요.

라고 하신 것을 감안하여

해당 label을 제거하고
/remove-label tide/merge-method-squash

수동으로 squash 를 하겠습니다. 😊

seokho-son commented:
감사합니다!
/lgtm
/approve
